### PR TITLE
Rewording predefined social and button text

### PR DESCRIPTION
**Pull request by t0p0lin0**
Please provide your thoughts/review on this important aspect of the site. Thanks to @kenman345 for the base suggestion that I tweaked ever so slightly. Needs to be finalized before launching.

### CODE REVIEW

**Comment by t0p0lin0** [comment]
It may be hard to parse via the code, but here's the button text:

Twitter: _Tweet to them_
Facebook: _Write on their wall_

...and the pre-defined Tweet text: _Please consider accepting #bitcoinCash for payments @above_domain - it’s fast and secure, with low fees! More @ https://acceptbitcoin.cash_

File: index.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: default
 
-link: Tell them to accept Bitcoin Cash
+link: Tweet to them
 link_mobile: TWEET THEM
-workonit_tweet: Please consider accepting Bitcoin Cash as a payment option @TWITTERHANDLE- it’s fast and secure, with low fees!
+workonit_tweet: "Please consider accepting #bitcoinCash for payments @TWITTERHANDLE - it’s fast and secure, with low fees! More @ "
```
Comment (zquestz) [inline index.html]: I would use #BitcoinCash

Comment (kenman345) [reply index.html]: Agreed, though I did write it originally with the lower case B, I am just used to camel casing it....

When we use this text it adds out url to it?

Comment (t0p0lin0) [reply index.html]: It does append our URL to the end, yep.

Comment (kenman345) [reply index.html]: Perfect then. Can we change it to #BitcoinCash and then I'd accept it.
